### DISCUSSION TITLE
Show TypeScript errors in development mode.

### DISCUSSION THREAD

**Discussion by SiddharthShyniben** [vercel/next.js#32591] [Ideas]
### Describe the feature you'd like to request

It would be great if next showed TypeScript errors when doing `next dev`. 

It's so unintuitive when a TypeScript error passes through local testing but fails on CI/CD just because I only started a local dev server.

### Describe the solution you'd like

Show TypeScript errors in dev mode (possibly show other ones too, I'm new to next)

### Describe alternatives you've considered

The only alternative I see is building locally. But that's what CI/CD is for.

**Comment by balazsorban44** [comment]
Hi, just curious, what code editor are you using? Modern IDEs like VSCode surface type errors right where you make them, and can even show you documentation links, etc. on hover.

**Reply by SiddharthShyniben** [reply]
I use Vim, and I do see the errors.

But I can't assume everyone has a "modern" IDE to show TypeScript errors. And sometimes you may not have an IDE open. 

There are situations where it would be helpful when the errors are in the output.
